buildSrc/src/test/resources/edu/ucar/build/ui/netCDFtoolsExtraJars.jnlp

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<jnlp spec="7.0" codebase="http://www.unidata.ucar.edu/software/thredds/current/netcdf-java/webstart"
+        href="netCDFtoolsExtraJars.jnlp" version="1.5">
+    <component-desc/>
+    
+    <resources>
+        <jar href='root-1.5.jar' main='true' download='eager' />
+    
+        <jar href='slf4j-api-1.7.7.jar' download='lazy' />
+        <jar href='objenesis-2.2.jar' download='lazy' />
+        <jar href='hamcrest-core-1.3.jar' download='lazy' />
```

Comment:
Is there any way to tie these dependency versions into what is used in `dependencies.gradle`? The `objenesis` dep in `dependencies.gradle` is version 2.4, and `slf4j` is 1.7.22, whereas the ones here are older. I can just imagine the dependencies in netCDFtoolsExtraJars.jnlp will get accidentally forgotten when we update other libraries in `dependencies.gradle`